Gemfile

```diff
@@ -1,5 +1,10 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
+gem 'jekyll'
```

Comment:
Ideally these versions should be locked, at least to a minor version:
```
gem 'jekyll''~>3.5.2'
```

so that the site doesn't completely fall apart if someone decides to update the Rubygems after a new major release of anything comes out.

but this is OK too. Thanks!